evennia/contrib/actions/setup.py

```diff
@@ -0,0 +1,20 @@
+from evennia import ObjectDB
```

Comment:
I suggest to remove this file and move this code out from here (and from `__init__.py` into utils.py.